hwdb/60-sensor.hwdb

```diff
@@ -160,6 +160,10 @@ sensor:modalias:acpi:BMA250E*:dmi:bvnINSYDECorp.:bvrG1D_S165*:svnilife:pnS165:*
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo:pnX1D3_C806N:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
 
+# Chuwi Hi10 (CWI1515)
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrP02A_C106.60E:bd03/18/2016:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnHampoo:rnCherryTrailCR:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:
```

Comment:
I'm pretty sure this line needs to be trimmed. It should not include the bd at least.
Maybe 
`sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrP02A_C106.60E:*:svnDefaultstring:pnDefaultstring:*`.

Reply:
Thanks for pointing this out. I will test it tomorrow evening.

Reply:
The trimmed identification line is working.

Added in https://github.com/systemd/systemd/pull/13351/commits/ac820765855a4f93730ce7bf4a6cfb0db60be4d3.